tests/trading/test_makeOrder.py

```diff
@@ -50,12 +55,12 @@ def test_setController_failure(contractsFixture):
 def test_publicMakeOrder_bid(contractsFixture):
     cash = contractsFixture.cash
     market = contractsFixture.binaryMarket
-    ordersFetcher = contractsFixture.contracts['ordersFetcher']
+    ordersFetcher = contractsFixture.contracts['NewOrdersFetcher']
     makeOrder = contractsFixture.contracts['makeOrder']
     cash.depositEther(value = 10**17)
     cash.approve(makeOrder.address, 10**17)
 
-    orderID = makeOrder.publicMakeOrder(BID, 10**18, 10**17, market.address, 1, 0, 0, 7)
+    orderID = makeOrder.publicMakeOrder(BID, 10**18, 10**17, market.address, 1, HASHED_ORDER_ID_0, HASHED_ORDER_ID_0, 7)
```

Comment:
I believe this should be a 20 byte array of 0s, not a hash of the number `0`.  In one of the other tests you correctly have `ZEROED_ORDER_ID = '\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'`, recommend using that here as well.

Reply:
Same comment throughout the test changes.  When we were passing in `0` for a better order ID or worse order ID it was meant to represent `null`, which would be a 20-byte null array.